src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (a) => {
+    return (b) => {
+      if (counter >= 3 && counter % 2 !== 0) {
+        counter++;
```

Comment:
If we do   `counter++;` in both conditions, let's just put it on line higher to write it only once